Circular_Linked_List/Circular_Linked_List.java

```diff
@@ -0,0 +1,135 @@
+class node{
+    int data;
+    node next;
+    
+    node(int data, node next){
+        this.data=data;
+        this.next=next;
```

Comment:
Please follow coding conventions throughout the code.
https://www.oracle.com/technetwork/java/javase/documentation/codeconventions-141388.html